k8s/crd/v16/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
```

Comment:
is there no way to use variables instead of copy pasting the same file multiple times?

Reply:
Figured out a way to use yq in GitHub Action to eliminate the copy-pasta